src/org/folio/rest_v2/Consortia.groovy

```diff
@@ -100,12 +102,43 @@ class Consortia extends Authorization {
         OkapiTenantConsortia centralConsortiaTenant = consortiaTenants.find { it.isCentralConsortiaTenant }
         createConsortia(centralConsortiaTenant)
         addCentralConsortiaTenant(centralConsortiaTenant)
-        // Required sleep for 2 minutes because consorita async installation
-        sleep(1 * 60 * 1000)
+        checkConsortiaStatus(centralConsortiaTenant, centralConsortiaTenant)
         consortiaTenants.findAll { (!it.isCentralConsortiaTenant) }.each { institutionalTenant ->
             addConsortiaTenant(centralConsortiaTenant, institutionalTenant)
-            // Required sleep for 2 minutes because consorita async installation
-            sleep(1 * 60 * 1000)
+            checkConsortiaStatus(centralConsortiaTenant, institutionalTenant)
+        }
+    }
+
+    /**
+     * Check consortia mapped tenants status
+     *
+     * @param centralConsortiaTenant
+     *
+     * @param tenant
+     *
+     */
+    void checkConsortiaStatus(OkapiTenantConsortia centralConsortiaTenant, OkapiTenantConsortia tenant){
+        Map headers = getAuthorizedHeaders(centralConsortiaTenant)
+        String endpoint = generateUrl("/consortia/${centralConsortiaTenant.consortiaUuid}/tenants/${tenant.tenantId}")
+        def response = restClient.get(endpoint, headers, 5000).body
+        switch (response['setupStatus']){
+            case 'COMPLETED':
+                logger.info("Tenant : ${tenant.tenantId} added successfully")
+                break
+            case 'COMPLETED_WITH_ERRORS':
+                logger.info("Tenant : ${tenant.tenantId} added with errors!")
+                break
+            case 'FAILED':
+                logger.error("Tenant : ${tenant.tenantId} add operation failed!\nAborting current execution!")
+                if (response['setupStatus'] == 'FAILED') {
```

Comment:
In this case, logger.error will throw an exception so this if will never be executed.
Also as you already in case 'FAILED' this if is not needed, better do

case 'FAILED':
steps.currentBuild.result = 'ABORTED'
logger.error("Tenant : ${tenant.tenantId} add operation failed!\nAborting current execution!")